tools/make_emoji.py

```diff
@@ -4,9 +4,10 @@
     print("pip install emoji")
     raise
 
-from emoji.unicode_codes import EMOJI_ALIAS_UNICODE
+from tkinter import W
+from emoji.unicode_codes import EMOJI_ALIAS_UNICODE_ENGLISH
 
-emoji = {k.lower().strip(":"): v for k, v in EMOJI_ALIAS_UNICODE.items()}
+emoji_dict = {k.lower().strip(":"): v for k, v in EMOJI_ALIAS_UNICODE_ENGLISH.items()}
```

Comment:
This seems to be a bug, typing seems to be right, there's no matching attribute in https://github.com/carpedm20/emoji/blob/master/emoji/unicode_codes/__init__.py that I can see. Mayne this was what was meant?

Also avoiding overriding the imported `emoji` module with a dict.

Reply:
That might be a fix. The code in tools isn't meant for public consumption. There's little point in maintaining code that is only use very occasionally by us.